README.md

```diff
@@ -1,29 +1,42 @@
 # TREK
 
-TREK is a web application that displays deals on travel packages. 
+TREK is a web application that displays deals on travel packages.
 
-You will build a single page application that will query data from an api, using AJAX, to display a list or trips and view more information on a specific trip. While viewing a specific trip, you can claim your spot by sending data back to the API. 
+This is a [stage 2](https://github.com/Ada-Developers-Academy/pedagogy/blob/master/rule-of-three.md) individual project.
 
-All of this should be done without ever having to reload the page. Meaning, it will be a single page application and will only need one html page (index.html). 
+## Learning Goals
 
-Reminder: You shall use HTML best-practices. You should organize your HTML by grouping content using semantic HTML tags.
+- Reading data from an API using AJAX
+- Posting data to an API using AJAX
+- Building dynamic elements on a page with jQuery
+- Using Underscore templates to simplify rendering HTML from JavaScript
 
-## The API
+## Description
+
+You will build a web app that will query data from an api, using AJAX, to display a list or trips and view more information on a specific trip. While viewing a specific trip, you can claim your spot by sending data back to the API.
 
-View the [Travel API's documentation](https://github.com/ada-c6/trip_api). 
+All of this should be done without ever having to reload the page. In other words, TREK will be a single page application and will only need one html page (index.html).
 
+## The API
 
-## Minimum Requirements 
-  - **Wave 1** Click button or link to show all trips
-  - **Wave 2** Can click on a trip, from the list of trips, to see more information about that trip
-    - Should be able to see id, name, destination, continent, about, category, weeks and cost 
-  - **Wave 3** While viewing a single trip, you can reserve a spot
-    - Use a form to submit your name to reserve a spot on the trip you are viewing
+View the [Travel API's documentation](https://github.com/ada-c6/trip_api).
```

Comment:
Should this link be updated to an Ada-C7 fork, or possibly just use the AdaGold repo?